provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitionerrole/PractitionerRoleTransformer.java

```diff
@@ -1,3 +1,94 @@
 package gov.va.api.health.providerdirectory.service.controller.practitionerrole;
 
-public class PractitionerRoleTransformer {}
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.allBlank;
+
+import gov.va.api.health.providerdirectory.api.datatypes.CodeableConcept;
+import gov.va.api.health.providerdirectory.api.datatypes.Coding;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.elements.Reference;
+import gov.va.api.health.providerdirectory.api.resources.PractitionerRole;
+import gov.va.api.health.providerdirectory.api.resources.PractitionerRole.PractitionerContactPoint;
+import gov.va.api.health.providerdirectory.service.PpmsPractitionerRole;
+import gov.va.api.health.providerdirectory.service.PpmsProviderSpecialtiesResponse;
+import gov.va.api.health.providerdirectory.service.ProviderContacts;
+import gov.va.api.health.providerdirectory.service.ProviderResponse;
+import gov.va.api.health.providerdirectory.service.controller.EnumSearcher;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Since we are injecting the base URL, this class is no longer stateless and `@Service` is not the appropriate Spring annotation. Change to `@Component`.

Reply:
On further reflection, I think it makes more sense for these transformer classes to be stateless functions. If we need the base URL here, instead of injecting it, include it on the `PpmsPractitionerRole` data being passed in.